### ISSUE TITLE
Update CONTRIBUTING.md to specify starting Redis

### ISSUE THREAD

**Issue by SeanPrashad** [Seneca-CDOT/telescope#134]
My Node version is `12.12.0` and my operating system is Mac OS Catalina `10.15.1 (19B88)`.

As a new individual to telescope, I was trying to setup the project using the steps listed under `Setup` found [here](https://github.com/Seneca-CDOT/telescope/blob/master/docs/CONTRIBUTING.md#environment-setup).

However, an error occurred after step 5 (`5. Run npm test`) which states that the `able to add a job to the feed queue` test failed due to `connect ECONNREFUSED 127.0.0.1:6379`.

After grepping the repo for the port `6379`, I realized that I had not started the Redis container using ~`docker-compose up -d`~ `docker-compose up -d redis`. It would be great to include that command in our `CONTRIBUTING.md` docs before `Run npm install`.

**Comment by SeanPrashad** [comment]
I'll be happy to submit a patch for this 🦧

**Comment by SeanPrashad** [comment]
As these are unit tests, it might be helpful to use [`redis-mock`](https://www.npmjs.com/package/redis-mock) to simulate a running instance of Redis to avoid this problem in the future.

**Comment by humphd** [comment]
#122 is related to this as well.  A lot of people are stumbling on this.  We should fix it in a few ways (docs, better error message, etc).

redis-mock is interesting.  We could consider this, but we're already running Redis in CI, so it's not critical.  Probably worth filing another issue on it at least.  The work in #71 could likely benefit from it in testing as well.
